plugins/modules/rds_param_group.py

```diff
@@ -170,8 +179,13 @@ def update_parameters(module, connection):
     apply_method = 'immediate' if module.params['immediate'] else 'pending-reboot'
     errors = []
     modify_list = []
-    parameters_paginator = connection.get_paginator('describe_db_parameters')
-    existing = parameters_paginator.paginate(DBParameterGroupName=groupname).build_full_result()['Parameters']
+    existing = {}
+    try:
+        _existing = _describe_db_parameters(connection, DBParameterGroupName=groupname)
+        if _existing:
+            existing = _existing['Parameters']
+    except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
+        module.fail_json_aws(e)
```

Comment:
```suggestion
        module.fail_json_aws(e, msg="Failed to describe existing parameter groups")
```

I'm not a big fan of using fail_json_aws without a message, I find it a little more friendly to at least explain what you were doing.

Reply:
@tremble Thanks for reviewing. Done.